### PR TITLE
[CI] Moved to GitHub actions

### PR DESCRIPTION
**Pull request by mnocon**
JIRA: https://issues.ibexa.co/browse/IBX-2314

I had to update php-cs-fixer, because it did not support PHP versions higher than 7.2.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    tests:
+        name: Unit tests
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 15
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '5.6'
```
Comment (alongosz) [inline .github/workflows/ci.yaml]: Why 5.6?

**Review by alongosz — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (mnocon) [reply .github/workflows/ci.yaml]: It was this way on Travis: https://github.com/ezsystems/ezplatform-design-engine/blob/2.0/.travis.yml#L9
and I didn't notice that it's unsupported by v2.5

I will remove it, thanks for noticing

Comment (mnocon) [reply .github/workflows/ci.yaml]: Removed in https://github.com/ezsystems/ezplatform-design-engine/pull/44/commits/acd6975c46a9f7831ef1f997b50a16ec572fabec

**Review by bogusez — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^5.7|^6.0",
-        "friendsofphp/php-cs-fixer": "~2.7.1",
+        "friendsofphp/php-cs-fixer": "^2.16.0",
```
Comment (alongosz) [inline composer.json:24]: Note: we need to switch to Ibexa Code Style here

File: composer.json [hunk 3]
```diff
@@ -36,7 +36,9 @@
         }
     },
     "scripts": {
-        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
+        "fix-cs": "php-cs-fixer fix -v --show-progress=estimating",
```
Comment (alongosz) [inline composer.json]: Afaik `estimating` is deprecated, there should be a warning coming from Fixer v2

File: bundle/DependencyInjection/Compiler/AssetThemePass.php [hunk 4]
```diff
@@ -31,7 +31,7 @@ public function process(ContainerBuilder $container)
         // Look for assets themes in bundles.
         foreach ($container->getParameter('kernel.bundles') as $bundleName => $bundleClass) {
             $bundleReflection = new \ReflectionClass($bundleClass);
-            $bundleViewsDir = dirname($bundleReflection->getFileName()) . '/Resources/public';
+            $bundleViewsDir = \dirname($bundleReflection->getFileName()) . '/Resources/public';
```
Comment (alongosz) [inline bundle/DependencyInjection/Compiler/AssetThemePass.php]: This requirement was either turned off by Ibexa Code style or forced to be in use directive

**Review by alongosz — CHANGES_REQUESTED** [review]

**Review by alongosz — APPROVED** [review]

**Review by ciastektk — APPROVED** [review]

**Pull request merged**